### PR TITLE
Increase snapshot history size

### LINKED ISSUE #224: Store more network map snapshots in netmap contract
According to [spec](https://github.com/nspcc-dev/neofs-api/blob/f6dad19bef26a049e05eaedfebf90852697c03a7/session/types.proto#L130-L138), Storage node should support `__NEOFS__NETMAP_EPOCH` and `__NEOFS__NETMAP_LOOKUP_DEPTH` x-headers for object service requests. 

It is not clear how we should implement it, because Storage nodes do not cache network maps and netmap contract contains only one previous epoch.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #224 , depends on #225 .

### CODE REVIEW

File: netmap/netmap_contract.go [hunk 1]
```diff
@@ -35,10 +35,13 @@ const (
 	notaryDisabledKey = "notary"
 	innerRingKey      = "innerring"
 
-	snapshot0Key     = "snapshotCurrent"
-	snapshot1Key     = "snapshotPrevious"
-	snapshotEpoch    = "snapshotEpoch"
-	snapshotBlockKey = "snapshotBlock"
+	// SnapshotCount contains the number of previous snapshots stored by this contract.
+	// Must be less than 255.
+	SnapshotCount        = 10
```
Comment (carpawell) [inline netmap/netmap_contract.go:40]: - may it be useful to be able to change that value?
- may it be useful to be able to get that value from the contract?

Comment (fyrchik) [reply netmap/netmap_contract.go:40]: Both yes.

File: netmap/netmap_contract.go [hunk 2]
```diff
@@ -396,7 +430,7 @@ func LastEpochBlock() int {
 // of current epoch.
 func Netmap() []storageNode {
 	ctx := storage.GetReadOnlyContext()
-	return getSnapshot(ctx, snapshot0Key)
+	return getSnapshot(ctx, snapshotKeyPrefix+"0")
```
Comment (alexvanin) [inline netmap/netmap_contract.go:433]: Shouldn't it be `snapshotKeyPreifx+storage.Get(snapshotCurrentIDKey)`?

File: netmap/netmap_contract.go [hunk 3]
```diff
@@ -414,18 +448,14 @@ func NetmapCandidates() []netmapNode {
...
 func Snapshot(diff int) []storageNode {
-	var key string
-
-	switch diff {
-	case 0:
-		key = snapshot0Key
-	case 1:
-		key = snapshot1Key
-	default:
+	if SnapshotCount <= diff {
```
Comment (alexvanin) [inline netmap/netmap_contract.go:451]: Negative `diff` should be invalid too.

File: netmap/netmap_contract.go [hunk 4]
```diff
@@ -85,6 +88,15 @@ func _deploy(data interface{}, isUpdate bool) {
 
 	if isUpdate {
 		common.CheckVersion(args.version)
+
+		data := storage.Get(ctx, "snapshotCurrent")
+		storage.Put(ctx, snapshotKeyPrefix+"0", data)
+
+		data = storage.Get(ctx, "snapshotPrevious")
+		storage.Put(ctx, snapshotKeyPrefix+"1", data)
```
Comment (alexvanin) [inline netmap/netmap_contract.go:96]: If I understand correctly, we put new epochs with incremented `id`. So previous snapshot should be with `id:0`, current snapshot with `id:1` and `snapshotCurrentIDKey:1`.

File: netmap/netmap_contract.go (hunk 2 above)
Comment (fyrchik) [reply netmap/netmap_contract.go:433]: Fixed.

File: netmap/netmap_contract.go (hunk 3 above)
Comment (fyrchik) [reply netmap/netmap_contract.go:451]: Fixed.

File: netmap/netmap_contract.go (hunk 4 above)
Comment (fyrchik) [reply netmap/netmap_contract.go:96]: Yes, fixed.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**